src/Dfc.CourseDirectory.Core/DataStore/CosmosDb/QueryHandlers/GetCourseTextByLearnAimRefHandler.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using Dfc.CourseDirectory.Core.DataStore.CosmosDb.Models;
+using Dfc.CourseDirectory.Core.DataStore.CosmosDb.Queries;
+using Microsoft.Azure.Documents.Client;
+using Microsoft.Azure.Documents.Linq;
+
+namespace Dfc.CourseDirectory.Core.DataStore.CosmosDb.QueryHandlers
+{
+    public class GetCourseTextByLearnAimRefHandler : ICosmosDbQueryHandler<GetCourseTextByLearnAimRef, CourseText>
+    {
+        public async Task<CourseText> Execute(DocumentClient client, Configuration configuration, GetCourseTextByLearnAimRef request)
+        {
+            var collectionUri = UriFactory.CreateDocumentCollectionUri(
+                configuration.DatabaseId,
+                configuration.CourseTextCollectionName);
+
+            var query = client.CreateDocumentQuery<CourseText>(collectionUri, new FeedOptions() { EnableCrossPartitionQuery = true })
+                .AsQueryable();
+
+            if (int.TryParse(request.LearnAimRef, out var learnAimRef))
```

Comment:
Because this: https://github.com/SkillsFundingAgency/dfc-providerportal-coursetext/blob/dev/src/Dfc.ProviderPortal.CourseText/Helpers/CosmosDbHelper.cs#L131

Reply:
Will fix when we migrate the `coursetext` collection to Sql Server